docs/config.md

```diff
@@ -132,16 +132,16 @@ profile_models: <comma-delimited-string-list>
 # Triton Docker image tag used when launching using Docker mode
 [ triton_docker_image: <string> | default: nvcr.io/nvidia/tritonserver:22.09-py3 ]
 
-# Triton Server HTTP endpoint url used by Model Analyzer client. Will be ignored if server-launch-mode is not 'remote'".
+# Triton Server HTTP endpoint url used by Model Analyzer client.".
```

Comment:
You're missing 3 more 'ignore' cases from config_command_profile, which is what feeds into the incorrect `--help` output

Reply:
Fixed